mteb/abstasks/AbsTaskInstructionRetrieval.py

```diff
@@ -326,205 +326,176 @@ def load_data(self, **kwargs):
-                corpus, queries = self.corpus[lang][split], self.queries[lang][split]
-                og_relevant_docs, changed_relevant_docs = (
-                    self.og_relevant_docs[lang][split],
-                    self.changed_relevant_docs[lang][split],
-                )
-                og_instructions, changed_instructions = (
-                    self.og_instructions[lang][split],
-                    self.changed_instructions[lang][split],
-                )
-
-                top_ranked = self.top_ranked[lang][split]
-                scores_og[lang], results_og[lang] = self._evaluate_subset(
-                    retriever,
-                    corpus,
-                    queries,
-                    og_relevant_docs,
-                    og_instructions,
-                    top_ranked,
-                    lang,
-                    **kwargs,
-                )
-                scores_changed[lang], results_changed[lang] = self._evaluate_subset(
-                    retriever,
-                    corpus,
-                    queries,
-                    changed_relevant_docs,
-                    changed_instructions,
-                    top_ranked,
-                    lang,
-                    **kwargs,
-                )
+        top_ranked = top_ranked[split]
+        scores_og, results_og = self._evaluate_subset(
+            retriever,
+            corpus,
+            queries,
+            og_relevant_docs,
+            og_instructions,
+            top_ranked,
+            lang,
+            **kwargs,
+        )
+        scores_changed, results_changed = self._evaluate_subset(
+            retriever,
+            corpus,
+            queries,
+            changed_relevant_docs,
+            changed_instructions,
+            top_ranked,
+            lang,
+            **kwargs,
+        )
 
-                newly_irrelevant_qrels = self.create_qrel_diff(
-                    self.og_relevant_docs[lang][split],
-                    self.changed_relevant_docs[lang][split],
-                )
-                overall_changed_scores[lang] = utils.evaluate_change(
-                    results_og[lang], results_changed[lang], newly_irrelevant_qrels
-                )
+        newly_irrelevant_qrels = self.create_qrel_diff(
+            og_relevant_docs,
+            changed_relevant_docs,
+        )
+        overall_changed_scores = utils.evaluate_change(
+            results_og, results_changed, newly_irrelevant_qrels
+        )
 
-                overall_changed_scores[lang]["individual"] = {
-                    "original": scores_og[lang],
-                    "changed": scores_changed[lang],
-                }
+        for k, v in scores_og.items():
+            overall_changed_scores[f"individual_original_{k}"] = v
+        for k, v in scores_changed.items():
+            overall_changed_scores[f"individual_changed_{k}"] = v
 
-                if self.do_length_ablation:
```

Comment:
Was this portion removed entirely? Or just moved to a different portion I can't see?

Reply:
I just moved